### PR TITLE
add migrations

### PR DESCRIPTION
**Pull request by kaallaash**

### CODE REVIEW

**Review by Blackolf931 — APPROVED** [review]

File: DescriptiveGeometry/DG.DAL/Entities/DrawingRow.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
-﻿namespace DG.DAL.Context.Entities;
+﻿namespace DG.DAL.Entities;
 
 public class DrawingRow
 {
     public int Id { get; set; }
     public string DrawingPhotoLink { get; set; }
-    public string DownLoadCount { get; set; }
+    public int DownLoadCount { get; set; }
```
Comment (nantonov) [inline DescriptiveGeometry/DG.DAL/Entities/DrawingRow.cs:7]: DownLoad => Download
And use plural form => Download**s**Count

**Review by nantonov — CHANGES_REQUESTED** [review]
Get rid of unnecessary migrations, create 1 clear migration instead

File: DescriptiveGeometry/DG.DAL/Registration.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using DG.DAL.Context;
+
+using Microsoft.EntityFrameworkCore;
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DG.DAL;
+
+public static class Registration
```
Comment (nantonov) [inline DescriptiveGeometry/DG.DAL/Registration.cs:10]: Wrong naming 
Registration is an action

File: DescriptiveGeometry/DG.DAL/Migrations/DatabaseContextModelSnapshot.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    partial class DatabaseContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "6.0.10")
+                .HasAnnotation("Relational:MaxIdentifierLength", 128);
+
+            SqlServerModelBuilderExtensions.UseIdentityColumns(modelBuilder, 1L, 1);
```
Comment (nantonov) [inline DescriptiveGeometry/DG.DAL/Migrations/DatabaseContextModelSnapshot.cs:23]: Is it needed?

File: DescriptiveGeometry/DG.DAL/Migrations/DatabaseContextModelSnapshot.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+{
+    [DbContext(typeof(DatabaseContext))]
+    partial class DatabaseContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "6.0.10")
+                .HasAnnotation("Relational:MaxIdentifierLength", 128);
```
Comment (nantonov) [inline DescriptiveGeometry/DG.DAL/Migrations/DatabaseContextModelSnapshot.cs:21]: Is it needed?

**Review by nantonov — CHANGES_REQUESTED** [review]

File: DescriptiveGeometry/DG.DAL/Migrations/DatabaseContextModelSnapshot.cs (hunk 3 above)
Comment (kaallaash) [reply DescriptiveGeometry/DG.DAL/Migrations/DatabaseContextModelSnapshot.cs:23]: I don't know how I fix it

**Review by Blackolf931 — APPROVED** [review]

**Review by nantonov — CHANGES_REQUESTED** [review]

File: DescriptiveGeometry/DG.DAL/Context/HistoryRepository.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace DG.DAL.Context;
+
+internal class HistoryRepository : SqlServerHistoryRepository
```
[the remark this answers is not in the document]
Comment (kaallaash) [reply DescriptiveGeometry/DG.DAL/Context/HistoryRepository.cs:13]: Sorry, I was inattentive

**Review by moltenessence — APPROVED** [review]

**Review by Blackolf931 — APPROVED** [review]

**Review by nantonov — APPROVED** [review]

**Pull request merged**